packages/apollo-server-azure-functions/src/__tests__/azureFunctionApollo.test.ts

```diff
@@ -25,9 +25,12 @@ const createAzureFunction = (options: CreateAppOptions = {}) => {
     req.on('data', chunk => (body += chunk));
     req.on('end', () => {
       const urlObject = url.parse(req.url, true);
+      try {
+        body = JSON.parse(body);
+      } catch (e) {}
```

Comment:
Should something be logged here for debugging purposes? Maybe at a low log level?

Reply:
I only saw one other place in the tests where console logs are being used and it seemed to be more for debugging a failed 'expect' validation.  I don't think it makes sense to add logging for a case that will always be hit for a specific test case

Reply:
Oh, I misunderstood what was happening. I did not realize this was a testcase. 🤦‍♂